### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by markjbrown**
## Changelog

* Initial commit. New template to support docs on integrating Cosmos & Web Apps service.
*
*

### CODE REVIEW

**Comment by markjbrown** [comment]
Hi @bmoore-msft this one is ready to go for merge.

Thanks.

**Comment by bmoore-msft** [comment]
Can you check the BP violations in the pipeline?

**Comment by markjbrown** [comment]
yeah just saw them. pushing new commit now to fix

**Comment by markjbrown** [comment]
I am not sure why the gov cloud test succeeded and the public cloud failed.

**Comment by markjbrown** [comment]
Closing and reopening to run the tests again.

**Comment by markjbrown** [comment]
@bmoore-msft. Ok, passed. there is a best practice issue for the reference call I make but it is the secret sauce for this template. Thanks.

File: 101-cosmosdb-webapp/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+- **Application Name:** Pick a name that will be used for the Cosmos account, App Plan and Web App.
+- **Location:** Azure region for all the resources.
+- **App Service Plan Tier:** App Service Plan's pricing tier.
+- **App Service Plan Instances:** App Service Plan's instance count.
+- **Repo URL:** The URL for the GitHub repository that contains the project to deploy.
+- **Branch:** The branch of the GitHub repository to use.
+
+[![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-cosmosdb-webapp%2Fazuredeploy.json)
+[![Deploy To Azure Gov](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-cosmosdb-webapp%2Fazuredeploy.json)
```
Comment (bmoore-msft) [inline 101-cosmosdb-webapp/README.md:26]: fix gov button

File: 101-cosmosdb-webapp/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  "resources": [
+    {
+      "type": "Microsoft.DocumentDB/databaseAccounts",
+      "name": "[variables('cosmosAccountName')]",
+      "apiVersion": "2020-04-01",
+      "kind": "GlobalDocumentDB",
+      "location": "[parameters('location')]",
+      "properties": {
+        "consistencyPolicy": {"defaultConsistencyLevel": "Session"},
+        "locations": [{"locationName": "[parameters('location')]", "failoverPriority": 0, "isZoneRedundant": false}],
```
Comment (bmoore-msft) [inline 101-cosmosdb-webapp/azuredeploy.json:80]: format the JSON (e.g. SHIFT+ALT+F in VS Code)

Comment (markjbrown) [reply 101-cosmosdb-webapp/azuredeploy.json:80]: ok, fixed in next commit.

File: 101-cosmosdb-webapp/README.md (hunk 1 above)
Comment (markjbrown) [reply 101-cosmosdb-webapp/README.md:26]: I can't see what is wrong with gov cloud button. Can you point to it?

Comment (markjbrown) [reply 101-cosmosdb-webapp/README.md]: found it. fixed in next commit

**Comment by markjbrown** [comment]
ok @bmoore-msft. All checks passed. Please merge ASAP if you can. Need to get a doc submitted for PR that depends on this one. Thanks again for your help on these.

**Pull request merged**